docs/design-structure-support/technical-assistance.md

```diff
@@ -3,58 +3,58 @@ sidebar_position: 1
 ---
 
 # How do I design and deliver technical assistance to DPGs?
-If the startup is considering open source or open source already it is still important to understand if they’ve covered all their bases including; 
-1. Should they even consider adopting an open source strategy in your startup? And; 
-2. Conduct a needs assessment on support the startup will need and; 
-3. Understanding their own resources as a startup and governance and; 
```

Comment:
This opening introduced a few things I couldn't find later on. I just removed them for clarity.